php-apache/5.6/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM quay.io/continuouspipe/ubuntu:16.04_v1
```

Comment:
Double check this is from the latest master, as it would be `quay.io/continuouspipe/ubuntu16.04:v1.0`